cmd/segment-reaper/detect.go

```diff
@@ -215,23 +93,8 @@ func cmdDetect(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
-func analyzeProject(ctx context.Context, db metainfo.PointerDB, objectsMap ObjectsMap, csvWriter *csv.Writer) error {
+func analyzeProject(ctx context.Context, db metainfo.PointerDB, projectID string, buckets bucketsObjects, csvWriter *csv.Writer) error {
```

Comment:
this can be a part of observer

Reply:
this will be used for analyze objects from `buckets bucketsObjects` and search for bad segments

Reply:
I'm ok with that. I haven't done that change because I didn't know if there was a reason for it and it isn't wrong to have just a separated function.

Reply:
done!